docs/src/main/sphinx/release/release-352.md

```diff
@@ -59,8 +59,9 @@
 * Match columns by name rather than by index by default for Parquet files. ({issue}`6479`)
 * Remove the `hive.partition-use-column-names` configuration property and the
   `partition_use_column_names ` session property. This is now determined automatically. ({issue}`6479`)
-* Support reading timestamp with microsecond or nanosecond precision (as configured with 
+* Support reading timestamps with microsecond or nanosecond precision (as configured with
   `hive.timestamp-precision` property) nested within `array`, `map` or `struct` data types. ({issue}`5195`)
+* Support writing timestamps with microsecond or nanosecond precision. ({issue}`3977`)
```

Comment:
I don't think there's a more-specific issue than #3977 to link here. There's more-specific issues for timestamps in arrays (#5195 and #6760), but not for writing in general.

Reply:
And if  we were to link to a PR or two, which ones would that be?